src/components/connection-modal/scanning-step.jsx

```diff
@@ -3,17 +3,21 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 
+import BalancedFormattedMessage from '../../containers/balanced-formatted-message.jsx';
 import Box from '../box/box.jsx';
 import PeripheralTile from './peripheral-tile.jsx';
 import Dots from './dots.jsx';
 
+import enterUpdateIcon from './icons/enter-update.svg';
```

Comment:
I took the liberty of running the new svg's through the tool svgo, which reduced their size about 30%